public/js/app/pages/__tests__/AboutCourseMemoEN.test.js

```diff
@@ -204,36 +202,39 @@ describe('User language: English. Component <AboutCourseMemo> show all memos: pd
     expect(menuItem.href).toBe('http://localhost/kursinfoadmin/kurser/kurs/KIP2720?l=en')
   })
 
-  xtest('renders "about memo" list, check pdf-based link as expected, user language', () => {
-    const listItem = getByText('Course memo KIP2720 Autumn 2019-23456')
+  test('renders "about memo" list, check pdf-based link as expected, user language', () => {
+    const listItem = getByText('Course memo KIP2720 Autumn 2019-2')
     expect(listItem).toBeInTheDocument()
     expect(listItem.href).toBe('http://localhost/kursinfostorage/memo-KIP272020192-14341833da79.pdf')
   })
 
-  xtest('renders "about memo" list, check web-based link as expected,  memo language', () => {
-    const listItem = getByText('Course memo KIP2720 Autumn 2019-12345')
+  test('renders "about memo" list, check web-based link as expected,  memo language', () => {
+    const listItem = getByText('Course memo KIP2720 Autumn 2019-1')
     expect(listItem).toBeInTheDocument()
-    expect(listItem.href).toBe('http://localhost/kurs-pm/KIP2720/KIP272020192-12345')
+    expect(listItem.href).toBe('http://localhost/kurs-pm/KIP2720/KIP272020192-1')
   })
 
-  xtest('renders "about memo" list, check web-based link as expected, memo language', () => {
-    const listItem = getByText('Course memo KIP2720 Autumn 2020-12345')
+  test('renders "about memo" list, check web-based link as expected, memo language', () => {
+    const listItem = getByText('Course memo KIP2720 Autumn 2020-1')
     expect(listItem).toBeInTheDocument()
-    expect(listItem.href).toBe('http://localhost/kurs-pm/KIP2720/KIP272020202-12345')
+    expect(listItem.href).toBe('http://localhost/kurs-pm/KIP2720/KIP272020202-1')
   })
 
   test('renders all links and check its number and labels', () => {
     const links = getAllByRole('link')
-    expect(links.length).toBe(9)
+    expect(links.length).toBe(12)
     const expectedlinks = [
       'Course and programme directory',
       'About course KIP2720',
       'Before course selection',
-      'Course memo Autumn 2019-12345',
-      'Course memo Autumn 2020-12345',
+      'Course memo Autumn 2019-1',
+      'Course memo Autumn 2020-1',
       'Course development',
       'Archive',
       'Administer About course',
+      'Course memo KIP2720 Autumn 2020-1',
```

Comment:
Where comes these three new links? Is it a new requirement

Reply:
We have not changed anything. I think page is not updated. If reload you can see the updated one. It's already in master.